Modules/+Sources/@VelocityLaser/VelocityLaser.m

```diff
@@ -211,6 +229,40 @@ function delete(obj)
                 rethrow(err)
             end
         end
+        
+        function val = set_wheel_ip(obj,val,~)
+            err = obj.connect_driver('wheel', 'ArduinoServo', val, obj.wheel_pin);
+            if isempty(obj.wheel)
+                if ~isempty(err)
+                    rethrow(err)
+                end
+                val = 'No Server';
+                return
+            end
+            if ~isempty(err)
+                rethrow(err)
+            end
+        end
+        function val = set_wheel_pin(obj,val,~)
+            err = obj.connect_driver('wheel', 'ArduinoServo', obj.wheel_ip, val);
+            if ~isempty(err)
+                rethrow(err)
+            end
+        end
+        function val = set_wheel_pos(obj,val,~)
+            if ~isempty(obj.wheel)
+    %             meta = obj.get_meta_pref('wheel_pos');
+
+                mask = cellfun(@(str)(strcmp(str, val)), obj.wheel_choices);
+                x = 1:length(mask);
+                index = x(mask);
+
+                assert(length(index) == 1);
+
+                obj.wheel.angle = 60 * (index-1);
```

Comment:
Can we remove the hardcoding to permit wheels with more or fewer slots?

Reply:
We can examine this issue if we ever hook up more than one Velocity.

Reply:
I agree with ebersin, let's toss a property and add an input to constructor to un-hardcode.